### PR TITLE
Commerce 2256

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
Hi @marco-leo , this PR applies the changes requested by @ethanbustad as well. Again please let me know about the commit messages.. Also I didn't made the change that manages the duplicated records for the tuples identified by "commerceInventoryWarehouseId"  and "sku"

File: commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/service/impl/CommerceInventoryWarehouseItemLocalServiceImpl.java [hunk 1]
```diff
@@ -56,36 +56,36 @@ public CommerceInventoryWarehouseItem addCommerceInventoryWarehouseItem(
...
 		if (Validator.isNotNull(sku)) {
-			CommerceInventoryWarehouseItem commerceInventoryWarehouseItem =
+			commerceInventoryWarehouseItem =
 				commerceInventoryWarehouseItemPersistence.fetchByC_S(
 					commerceInventoryWarehouseId, sku);
 
 			if (commerceInventoryWarehouseItem != null) {
 				throw new DuplicateCommerceInventoryWarehouseItemException(
-					"Duplicated sku code");
+					"Sku code already associated to this Warehouse");
```
Comment (ethanbustad) [inline commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/service/impl/CommerceInventoryWarehouseItemLocalServiceImpl.java]: should be `"...associated with..."`, not `"...associated to..."`

**Review by ethanbustad — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
Hi @ethanbustad , I've just fixed the typo you told me. BTW Thank you!

**Comment by ethanbustad** [comment]
ci:test
tests should be fixed in upstream now

**Review by ethanbustad — APPROVED** [review]

**Pull request merged**